### DISCUSSION TITLE
SD3.5 merge? I wonder when?

### DISCUSSION THREAD

**Discussion by VeteranXT** [lllyasviel/stable-diffusion-webui-forge#2278] [General]
I was just wondering when, I don't expect any ETA. Just by time we get SD3.5 new models will come out 💯

**Comment by thiagojramos** [comment]
Soon

**Comment by DIGITAL-SURFER** [comment]
Having SD 3.5 in Forge would certainly be a valuable extra.  Forge is getting close to blowing every other UI into oblivion the way it's going.

Incidentally, another program that the creator of Forge developed - Fooocus, already has a couple of branches that support SD3.

**Reply by andy8992** [reply]
While that's true, 3.5 support is essential for me. I need medium. It works very well on my PC. I really want to see it supported, more software support = more community support.

I really would like this to replace XL for me, but if it's never supported adoption of the model is gonna get hurt imo.

**Reply by VeteranXT** [reply]
Currently using SD3.5 in comfyUI on zluda. I just really do not like Tech Demo UI.
